tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -453,6 +464,7 @@ IncludeByKey.PASSTHROUGH {
 			{
 				index	6
 				direction	playback
+				core_id		$SSP2_PCM_CORE_ID
```

Comment:
@RanderWang the suggested change is fine for the regular nocodec topology. But for the passthrough nocodec topology, I'd suggest to keep all core IDs as 0. @plbossart what do you think?

Reply:
sure, I can image it. Removed.